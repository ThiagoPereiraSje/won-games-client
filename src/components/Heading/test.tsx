import { render, screen } from '@testing-library/react'
import Heading from '.'

describe('<Heading />', () => {
  it('should render the heading', () => {
    render(<Heading />)

    const element = screen.getByRole('heading', { name: /Heading/i })

    expect(element).toBeInTheDocument()
  })

  it('should render correctly', () => {
    const { container } = render(<Heading />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
