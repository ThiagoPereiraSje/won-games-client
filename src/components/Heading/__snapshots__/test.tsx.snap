// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> should render correctly 1`] = `
<main
  class=""
>
  <h1>
    Hello Heading
  </h1>
</main>
`;
